backend/api/application.py

```diff
@@ -23,13 +25,13 @@
     "description": "Create, update, delete, and retrieve TA Applications.",
 }
 
-api = APIRouter(prefix="/api/applications")
+api = APIRouter(prefix="/api/ta/applications")
```

Comment:
Super tiny nit: I wonder if `/api/applications/ta` might be better here, since we could see `/api/applications/ambassador` in the future!